Tests/PlistPropertyLoaderTests.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  PlistPropertyLoaderTests.swift
+//  Swinject
+//
+//  Created by mike.owens on 12/6/15.
+//  Copyright © 2015 Swinject Contributors. All rights reserved.
+//
+
+import XCTest
+import Swinject
+import SwinjectPropertyLoader
+
+
+class PlistPropertyLoaderTests: XCTestCase {
+    func testMissingResourcesAreHandled() {
+        let loader = PlistPropertyLoader(bundle: Bundle(for: type(of: self).self), name: "noexist")
+        XCTAssertThrowsError(try loader.load()) { error in
+            XCTAssert(error is PropertyLoaderError)
+        }
+    }
+    
+    // No test for invalid since Xcode won't allow invalid plist files to be added to the bundle
```

Comment:
Could we trick Xcode by using a different extension, then changing the extension in the test?

Reply:
Not sure, but it might be possible. We can try later.

I just left the same comment as the previous test using Quick and Nimble.
https://github.com/Swinject/SwinjectPropertyLoader/pull/24/files/df6d14071a205eddaa4fa704919e23a34b3d658c#diff-ae0b5a7a5f589366e54306d21b9176fd620b3362d5d4264db6abfef236fe03adL23